hyo2/soundspeed/formats/readers/sea_and_sun.py

```diff
@@ -15,6 +15,13 @@ class SeaAndSun(AbstractTextReader):
     Info: https://sea-sun-tech.com/technology.html
     """
 
+    # A dictionary to resolve probe type from probe type token
+    probe_dict = {
+        'CTD': Dicts.probe_types['SST-CTD'],
+        'CTP': Dicts.probe_types['SST-CTP'],   # deprecated
+        'CTM': Dicts.probe_types['SST-MEM']
```

Comment:
What is the naming rule here? How can 'SST-MEM' be 'CTM'?

Reply:
I've uploaded an excerpt from the Sea&Sun documentation (sorry it's only in German):

![NamingConvention](https://github.com/hydroffice/hyo2_soundspeed/assets/5863999/eceef906-3b11-4110-b0a4-cd177f9adbfc)

CTMxxxx ist what the TOB will contain in the header section of a TOB file. The xxxx will be replaced by the probe's S/N. CTM refers to a memory probe, which has the more correct designation SST-MEM (SST for Sea & Sun Technology) and MEM for (obviously) memory.